wpilibNewCommands/src/main/native/include/frc2/command/TrapezoidProfileSubsystem.h

```diff
@@ -37,18 +37,21 @@ class TrapezoidProfileSubsystem : public SubsystemBase {
    * when the subsystem is constructed.
    * @param period          The period of the main robot loop, in seconds.
    */
-  TrapezoidProfileSubsystem(Constraints constraints, Distance_t position,
+  TrapezoidProfileSubsystem(Constraints constraints,
```

Comment:
Needs to be made explicit.

Reply:
Is there a Java equivalent of this change?

Reply:
No; the Java has no default because it'd involve overloading the constructor.  It's a convenience feature that's worth it for C++ but probably not for the Java.

Reply:
We do that lots of other places though.  Overloading the constructor is fine.